tests/unit/restapi/lib/actions.py

```diff
@@ -101,3 +102,32 @@ def register_queue(
         json=payload,
         follow_redirects=True,
     )
+
+
+def register_tag(
+    client: FlaskClient,
+    name: str,
+    group_id: int | None = None,
```

Comment:
group_id should be be required. I checked and this is actually missing from the request schema. Can you add it there as well?

Reply:
Added with commit: https://github.com/usnistgov/dioptra/pull/455/commits/d89f7da3dbebebb37ff6f2a189405d92ae74d803